src/components/donateabox/index.tsx

```diff
@@ -2,11 +2,12 @@ import { createElement, FunctionComponent } from 'react'
 import { classNames } from 'utils'
 import styles from './style.scss'
 
-const DonateABox: FunctionComponent<{ char: string, className?: string }> = ({ char, className }) => {
+const DonateABox: FunctionComponent<{ char: string, fontSize: string, className?: string }> =
+({ char, fontSize, className }) => {
```

Comment:
``` diff
- const DonateABox: FunctionComponent<{ char: string, fontSize: string, className?: string }> =
- ({ char, fontSize, className }) => {
+ const DonateABox: FunctionComponent<{ char: string, fontSize: number, className?: string }> =
+ ({ char, fontSize = 110, className }) => {
```
If you are going to be adding a font size set the default to 110 and make it a number